apps/mobile/src/components/topbar/topbar.tsx

```diff
@@ -1,15 +1,28 @@
-import { Ionicons } from '@expo/vector-icons';
+import { Ionicons, MaterialIcons } from '@expo/vector-icons';
 import { useNavigation } from '@react-navigation/native';
-import { Box, Icon, Pressable } from 'native-base';
+import { t } from 'i18next';
+import { Box, Button, Icon, Pressable, Text } from 'native-base';
+
+import Routes from '@/routes';
 
 const Topbar = () => {
-  const { goBack } = useNavigation();
+  const { goBack, navigate } = useNavigation();
 
   return (
-    <Box display="flex" flexDirection="row">
+    <Box display="flex" flexDirection="column" alignItems="flex-start">
       <Pressable onPress={() => goBack()}>
         <Icon as={Ionicons} name="chevron-back-outline" size="lg" />
       </Pressable>
+
+      <Box mt={10}>
+        <Button
+          onPress={() => navigate(Routes.MyAdoptions as never)}
+          variant="ghost"
+          leftIcon={<Icon as={MaterialIcons} name="pets" size="lg" color="primary.300" />}
+        >
+          <Text color="primary.300">{t('MY_PETS_SCREEN.TITLE')}</Text>
+        </Button>
+      </Box>
```

Comment:
this component is supposed to be reused across the app screens meaning that you can't have a fixed title here. i'd say to receive a `title` prop and then conditionally render all this block